### ISSUE TITLE
NetworkError: 403 Forbidden

### ISSUE THREAD

**Issue by Claudeblet** [eagerterrier/MimeTypes-Link-Icons#29]
Hello,

I have a problem with the MimeTypes Link Icons extension. In the Firefox console, I get the following error:

`"NetworkError:` 403 Forbidden - `https://www.famillecorbet.fr/wp-content/plugins/mimetypes-link-icons/css/style.php?cssvars=bXRsaV9oZWlnaHQ9MTI4Jm10bGlfaW1hZ2VfdHlwZT1wbmcmbXRsaV9sZWZ0b3JyaWdodD1sZWZ0&ver=3.0"`

I now have this error:

`La` ressource à l’adresse « https://www.famillecorbet.fr/wp-content/plugins/mimetypes-link-icons/css/style.php?cssvars=bXRsaV9oZWlnaHQ9MTI4Jm10bGlfaW1hZ2VfdHlwZT1wbmcmbXRsaV9sZWZ0b3JyaWdodD1sZWZ0&ver=3.0 » a été bloquée en raison d’un type MIME incorrect (X-Content-Type-Options: `nosniff).`

Can you help me ?

Thank you in advance,

Regards,

Claude

**Comment by jrfnl** [comment]
Hi Claude,

Just checking: are you using the latest release as posted on wordpress.org or the `master` branch of this repo ?

If the latest release, you could try to add the `.htaccess` file from PR #24 to your install to see if that solves the problem. It seems it's missing from the release.

**Comment by Claudeblet** [comment]
Hello,

I used the last version. I will try to add the rules to the .htaccess file (PR # 24).

thanks again

Claude

**Comment by eagerterrier** [comment]
strange - there were a few missing files on that release

Have updated main w/p rev to 3.2.7

thanks all!

**Comment by sparky672** [comment]
I have the latest version of the plugin and it includes the `.htaccess` file, which contains the code as shown:

```
<Files style.php>
	Order Allow,Deny
	Allow from all
</Files>
```

I still have this console error:

>  https://{mydomain}/wp-content/plugins/mimetypes-link-icons/css/style.php?cssvars=bXRsaV9oZWlnaHQ9NjQmbXRsaV9pbWFnZV90eXBlPXBuZyZtdGxpX2xlZnRvcnJpZ2h0PWxlZnQ&ver=3.0
>  [Error] Failed to load resource: the server responded with a status of 403 (Access Forbidden) (style.php, line 0)

**Comment by jrfnl** [comment]
@sparky672 On what type of server are you running the code ? and if Apache, what version ?

**Comment by sparky672** [comment]
Yes, Apache Linux.  I do not have access to server os, but `phpinfo` reveals this...

>  Linux cubs.unisonplatform.com 2.6.32-042stab127.2 #1 SMP Thu Jan 4 16:41:44 MSK 2018 x86_64

**Comment by manzowa** [comment]
By the way, there is no sequel. Where is the solution?
